meganav.yaml

```diff
@@ -305,66 +308,53 @@ products:
             - title: Kubeflow installation tutorial
               url: https://charmed-kubeflow.io/docs/install
 
-    - title: Charmhub and Juju
+    - title: AI and data
       primary_links:
         - links:
-            - title: Charmhub
-              description: Store for Juju operators
-              url: https://charmhub.io/
-            - title: Juju
-              description: Orchestrator engine for operators
-              url: https://juju.is/
-            - title: JAAS
-              description: Auditing and compliance for Juju
-              url: https://jaas.ai/
+            - title: AI/ML
+              description: AI and machine learning on Ubuntu
+              url: /ai
+            - title: Kubeflow
+              description: End-to-end MLOps platform
+              url: https://charmed-kubeflow.io/
+            - title: MLflow
+              description: Lightweight ML platform
```

Comment:
```suggestion
              description: Lightweight machine learning platform
```
Is this in the doc

Reply:
Aaaah thank you for flagging this up. "Lightweight machine learning platform" doesn't fit and I agreed on "Lightweight ML platform" with Andreea, but forgot to reflect it in the copy doc, it's fixed now.